app/src/main/java/com/imashnake/animite/features/home/Home.kt

```diff
@@ -36,19 +36,17 @@ import com.imashnake.animite.core.extensions.landscapeCutoutPadding
 import com.imashnake.animite.core.ui.ProgressIndicator
 import com.imashnake.animite.core.ui.TranslucentStatusBarLayout
 import com.imashnake.animite.data.Resource
-import com.imashnake.animite.features.destinations.MediaPageDestination
-import com.imashnake.animite.features.media.MediaPageArgs
+import com.imashnake.animite.features.route.AnimiteRoute
 import com.imashnake.animite.features.ui.MediaSmall
 import com.imashnake.animite.features.ui.MediaSmallRow
 import com.ramcosta.composedestinations.annotation.Destination
-import com.ramcosta.composedestinations.navigation.DestinationsNavigator
 import com.imashnake.animite.R as Res
 
 @Destination
 @Composable
 fun Home(
     viewModel: HomeViewModel = hiltViewModel(),
-    navigator: DestinationsNavigator
+    onNavigateToMediaPage: (AnimiteRoute.MediaPage) -> Unit,
```

Comment:
Would it make more sense to just take the media ID here? Type will probably be available at a higher level than this (I think)

e.g.

```suggestion
    onNavigateToMediaPage: (id: Int) -> Unit,
```

Reply:
I noticed that the `mediaType` is always be `homeMediaType.rawValue`, so yeah it makes sense more to just pass the ID instead.

Reply:
No the type is only available in Home composable

Reply:
cc @imashnake0 is that intentional, or is that only because we can't change the media type yet?

Reply:
We don't change the media type yet because the lists are different for manga. I'm okay with doing either but know that we should eventually be able to change the type in the UI.

Reply:
Maybe pass the ID and hardcode MediaType.ANIME for now?

Reply:
Better to leave it as it is for now i guess